### DISCUSSION TITLE
How I can start sandbox, the akash throw error.

### DISCUSSION THREAD

**Discussion by taoyonggang** [akash-network/community#547] [Q&A]
akash start
INF initializing upgrade `v0.20.0`
INF initializing upgrade `v0.26.0`
INF initializing upgrade `v0.28.0`
INF initializing upgrade `v0.30.0`
INF initializing upgrade `v0.32.0`
INF initializing upgrade `v0.24.0`
INF initializing upgrade `akash_v0.15.0_cosmos_v0.44.x`
INF starting node with ABCI Tendermint in-process
INF service start impl=multiAppConn module=proxy msg={}
INF service start connection=query impl=localClient module=abci-client msg={}
INF service start connection=snapshot impl=localClient module=abci-client msg={}
INF service start connection=mempool impl=localClient module=abci-client msg={}
INF service start connection=consensus impl=localClient module=abci-client msg={}
INF service start impl=EventBus module=events msg={}
INF service start impl=PubSub module=pubsub msg={}
INF service start impl=IndexerService module=txindex msg={}
INF ABCI Handshake App Info hash= height=0 module=consensus protocol-version=0 software-version=v0.32.3
INF ABCI Replay Blocks appHeight=0 module=consensus stateHeight=0 storeHeight=0
INF asserting crisis invariants inv=1/11 module=x/crisis name=gov/module-account
INF asserting crisis invariants inv=2/11 module=x/crisis name=staking/module-accounts
INF asserting crisis invariants inv=3/11 module=x/crisis name=staking/nonnegative-power
INF asserting crisis invariants inv=4/11 module=x/crisis name=staking/positive-delegation
INF asserting crisis invariants inv=5/11 module=x/crisis name=staking/delegator-shares
INF asserting crisis invariants inv=6/11 module=x/crisis name=bank/nonnegative-outstanding
INF asserting crisis invariants inv=7/11 module=x/crisis name=bank/total-supply
INF asserting crisis invariants inv=8/11 module=x/crisis name=distribution/nonnegative-outstanding
INF asserting crisis invariants inv=9/11 module=x/crisis name=distribution/can-withdraw
INF asserting crisis invariants inv=10/11 module=x/crisis name=distribution/reference-count
INF asserting crisis invariants inv=11/11 module=x/crisis name=distribution/module-account
INF asserted all invariants duration=1.050045 height=0 module=x/crisis
INF created new capability module=ibc name=ports/transfer
INF port binded module=x/ibc/port port=transfer
INF claimed capability capability=1 module=transfer name=ports/transfer
panic: unknown field "deployment_min_deposit" in v1beta3.Params

goroutine 1 [running]:
github.com/cosmos/cosmos-sdk/codec.(*ProtoCodec).MustUnmarshalJSON(0xc000e068d0?, {0xc0000364e0?, 0x21d29e0?, 0xc000e068d0?}, {0x2c16d50?, 0xc000e06d80?})
	github.com/cosmos/cosmos-sdk@v0.45.16/codec/proto_codec.go:178 +0x3b
github.com/akash-network/node/x/deployment.AppModule.InitGenesis({{{0x2c41650, 0xc0005e8d70}}, {0x2c43660, 0xc00047f680}, {0x7f2d28355700, 0xc00047f710}, {0x7f2d28355728, 0xc00118ce80}, {0x2c4e370, 0xc0002c5b80}, ...}, ...)
	github.com/akash-network/node/x/deployment/module.go:180 +0x94
github.com/cosmos/cosmos-sdk/types/module.(*Manager).InitGenesis(_, {{0x2c2c178, 0x3f95200}, {0x2c3b828, 0xc0000669c0}, {{0x0, 0x0}, {0xc00111a990, 0xa}, 0x0, ...}, ...}, ...)
	github.com/cosmos/cosmos-sdk@v0.45.16/types/module/module.go:327 +0x227
github.com/akash-network/node/app.(*AkashApp).InitChainer(_, {{0x2c2c178, 0x3f95200}, {0x2c3b828, 0xc0000669c0}, {{0x0, 0x0}, {0xc00111a990, 0xa}, 0x0, ...}, ...}, ...)
	github.com/akash-network/node/app/app.go:525 +0x1ee
github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).InitChain(0xc0002af6c0, {{0x18791a26, 0xedc38d89d, 0x0}, {0xc00111a990, 0xa}, 0xc000338ca0, {0x3f95200, 0x0, 0x0}, ...})
	github.com/cosmos/cosmos-sdk@v0.45.16/baseapp/abci.go:63 +0x475
github.com/tendermint/tendermint/abci/client.(*localClient).InitChainSync(0xc0005f2540, {{0x18791a26, 0xedc38d89d, 0x0}, {0xc00111a990, 0xa}, 0xc000338ca0, {0x3f95200, 0x0, 0x0}, ...})
	github.com/tendermint/tendermint@v0.34.27/abci/client/local_client.go:272 +0x102
github.com/tendermint/tendermint/proxy.(*appConnConsensus).InitChainSync(0xc00014e3f0?, {{0x18791a26, 0xedc38d89d, 0x0}, {0xc00111a990, 0xa}, 0xc000338ca0, {0x3f95200, 0x0, 0x0}, ...})
	github.com/tendermint/tendermint@v0.34.27/proxy/app_conn.go:77 +0x55
github.com/tendermint/tendermint/consensus.(*Handshaker).ReplayBlocks(_, {{{0xb, 0x0}, {0x22c73b6, 0x7}}, {0xc00111a990, 0xa}, 0x1, 0x0, {{0x0, ...}, ...}, ...}, ...)
	github.com/tendermint/tendermint@v0.34.27/consensus/replay.go:319 +0xe98
github.com/tendermint/tendermint/consensus.(*Handshaker).Handshake(0xc000febc48, {0x2c3e168, 0xc000416410})
	github.com/tendermint/tendermint@v0.34.27/consensus/replay.go:268 +0x3a8
github.com/tendermint/tendermint/node.doHandshake({_, _}, {{{0xb, 0x0}, {0x22c73b6, 0x7}}, {0xc00111a990, 0xa}, 0x1, 0x0, ...}, ...)
	github.com/tendermint/tendermint@v0.34.27/node/node.go:329 +0x199
github.com/tendermint/tendermint/node.NewNode(0xc000661180, {0x2c13600, 0xc000fc0fa0}, 0xc000e0c230, {0x2c08740, 0xc000e163a8}, 0x0?, 0x0?, 0xc000feca30, {0x2c2c108, ...}, ...)
	github.com/tendermint/tendermint@v0.34.27/node/node.go:779 +0x557
github.com/cosmos/cosmos-sdk/server.startInProcess(_, {{0x0, 0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, {0x2c31850, 0xc0005fc1c0}, ...}, ...)
	github.com/cosmos/cosmos-sdk@v0.45.16/server/start.go:280 +0x897
github.com/cosmos/cosmos-sdk/server.StartCmd.func2(0xc000fc3200?, {0x3f95200?, 0x0?, 0x0?})
	github.com/cosmos/cosmos-sdk@v0.45.16/server/start.go:128 +0x1e8
github.com/spf13/cobra.(*Command).execute(0xc000fce300, {0x3f95200, 0x0, 0x0})
	github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
github.com/spf13/cobra.(*Command).ExecuteC(0xc0003d8f00)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	github.com/spf13/cobra@v1.7.0/command.go:985
github.com/akash-network/node/cmd/akash/cmd.Execute(0x3ebb848?, {0x22c4210, 0x5})
	github.com/akash-network/node/cmd/akash/cmd/root.go:113 +0x1e8
main.main()
	github.com/akash-network/node/cmd/akash/main.go:17 +0x25

**Comment by taoyonggang** [comment]
I am using version v0.32.3

**Comment by brewsterdrinkwater** [comment]
This issue was addressed in Akash's discord: https://discord.com/channels/747885925232672829/1111699100463792198/1229410983991181414

**Reply by challanger244** [reply]
> This issue was addressed in Akash's discord: https://discord.com/channels/747885925232672829/1111699100463792198/1229410983991181414

ayo this discord link isnt visable can we get a copy paste as this is also a single google result

**Reply by brewsterdrinkwater** [reply]
Akash's Discord is a builder focused communication hub for all technical related questions. Akash users who have technical questions should ask in discord for fast responses from Akash Insiders, Akash Vanguards, core team members, and many other community members.

The discord link above links to the question around sandbox and documentation that can be found here: https://akash.network/docs/akash-development-environment/overview-and-requirments/

**Reply by jmikedupont2** [reply]
I have the same problem using the git version of the code i just build.

v0.34.1

here is my startup trace 
https://gist.github.com/jmikedupont2/f6d3a4584c855e94552c0dda59858727

important snippet:
```
INF port binded module=x/ibc/port port=transferINF claimed capability capability=1 module=transfer name=ports/transfer

panic: unknown field "deployment_min_deposit" in v1beta3.Params 

goroutine 1 [running]:github.com/cosmos/cosmos-sdk/codec.(*ProtoCodec).MustUnmarshalJSON(0xc0034a8780?, {0xc00005fe00?, 0x21fc640?, 0xc0034a8780?}, {0x2c452c0?, 0xc0034a88a0?})	

github.com/cosmos/cosmos-sdk@v0.45.16/codec/proto_codec.go:178 +0x3b
```
i have discord but this link is taking forever to load,
what is the solution here for the record?

**Reply by Zblocker64** [reply]
Did you download the snapshot?
